tools/destop-app/build.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 
 # Build OpenMapper Desktop App
+
+cd wrapper/desktop/
 mkdir -p da_build
-cd build
-cmake ../wrapper/desktop-app  -DCMAKE_BUILD_TYPE=DEBUG 
-make -j8
+cd da_build
+cmake .. -DCMAKE_BUILD_TYPE=DEBUG 
+make -j4
```

Comment:
so you prefer a build folder in the wrapper/desktop dir?

this is fine for me, but then i would call it simply build, as it is clear for what target it is..